### DISCUSSION TITLE
where are the exe and panels?

### DISCUSSION THREAD

**Discussion by kims1000** [RomanKubiak/ctrlr#384] [General]
plz go back to ctrlr.org, i cant find anything here, like the exe or the panels - where are they??

**Comment by dobo365** [comment]
I personally also prefer the old site regarding discussion topics, downloads and panels catalogue but this is not our choice so we just follow.
From what I understood, some of the reasons for leaving the old site is the amount of spam and the fact that Wordpress is a bit capricious.
To find the code: click on Code at the left of the github menu appearing on pages then select Releases on the right side of that page.
But for older versions (like the good 5.3.201) I think you must still go to the old site.

**Comment by kims1000** [comment]
thank you, i think i found it...down in the jungle of the submenu´s here on github, so from Ctrlr-6.0.26 to Ctrlr-5.6.0...???

**Reply by dobo365** [reply]
Yes..just a mess.
Until we can clearly see a new version running without bugs, I keep using 5.3.201 and 5.3.198.
Not having any issue with those and I'm doing some tricky things.
When I'll find the time, I will try one of those new versions but...which one indeed? 
5.5.9 is indicated as being the latest. [(https://github.com/RomanKubiak/ctrlr/releases)](https://github.com/RomanKubiak/ctrlr/releases)

**Reply by Godlike-Productions** [reply]
I'm doing my final builds on 5.6.23 that Sjoerd compiled.  The reasons being
1) They are resizable
2) Menu is hidden
3) There is a build that supports iLok
4) There is a build for OSX and Windows that is the same.

Downsides is that when it crashes you don't get errors, so I'm developing with 5.3.201

**Reply by bijlevel** [reply]
I'm using now the 5.6.23 version for a couple of weeks (or even months) and it did crash only once... Also I think the UI is a bit more consistent and does solve some (minor) issues, like exporting with the right window size, so you don't need to add an extra 20 pix. I noticed Roman has recently uploaded a new source which seems to address the VST (the "classic" vestion) compiling for Mac. Indeed I was not able to compile the VST part successfully. (VST3 does succeed though. )If I find the time I will try and compile the latest source. I will let you know if I succeed of course!

**Reply by Godlike-Productions** [reply]
One small thing with the vst3's is that exported panels don't appear under Ableton Live.  It may be because it is marked dirty (it's a setting somewhere),or possibly because it's not signed. I'm not sure

**Reply by damiensellier** [reply]
@Godlike-Productions @bijlevel same here, I can't get MacOS exported panels as VST3 to appear in cubase. I tried all releases and even built my own, without success while exporting vst3 instances. 
EDIT: I went through the source files and found something that can help to fix the issue.  Please check #394 394

**Comment by Godlike-Productions** [comment]
I am also hosting some alternative builds that solve a number of problems here

https://godlike.com.au/index.php?id=361

**Comment by dobo365** [comment]
Anyone knowing where to find a .exe of the latest 5.5.9 as 5.6 is pre-release.
There is only a .sh file that installs a directory and some files but no .exe... (Windows).
Thx
